src/foam/mlang/mlangJava.js

```diff
@@ -217,6 +238,10 @@ foam.CLASS({
                 + '  if ( ! getArgs()[i].f(obj) ) return false;\n'
                 + '}\n'
                 + 'return true;'
+    },
+    {
+      name: 'toSQL',
+      javaCode: 'return null;'
```

Comment:
yeah, its just a place holder for now, will add  a TODO, otherwise would java wouldnt compile because the classes are not meeting the interface methods.